graviti/manager/dataset.py

```diff
@@ -5,25 +5,42 @@
 
 """The implementation of the Dataset and DatasetManager."""
 
-from typing import Any, Dict, Generator, Iterator, KeysView, Mapping, Optional, Tuple, Type, TypeVar
+from functools import partial
+from typing import (
+    Any,
+    Dict,
+    Generator,
+    Iterator,
+    KeysView,
+    List,
+    Mapping,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+)
+
+from tensorbay.utility import AttrsMixin, attr, common_loads
 
-from tensorbay.dataset import Notes, RemoteData
-from tensorbay.label import Catalog
-from tensorbay.utility import URL, AttrsMixin, attr, common_loads
-
-from graviti.client import get_catalog, get_notes, list_data_details, list_segments
 from graviti.dataframe import DataFrame
 from graviti.exception import ResourceNotExistError
 from graviti.manager.branch import BranchManager
 from graviti.manager.commit import CommitManager
 from graviti.manager.draft import DraftManager
 from graviti.manager.lazy import PagingList
 from graviti.manager.tag import TagManager
-from graviti.openapi import create_dataset, delete_dataset, get_dataset, list_datasets
-from graviti.portex import Extractors, catalog_to_schema, get_extractors
+from graviti.openapi import (
+    create_dataset,
+    delete_dataset,
+    get_dataset,
+    get_schema,
+    list_commit_data,
+    list_datasets,
+    list_sheets,
+)
 from graviti.utility import LazyFactory, LazyList, NestedDict, ReprMixin, ReprType
 
-LazyLists = NestedDict[str, LazyList[Any]]
+LazyLists = NestedDict[str, LazyList]
```

Comment:
Paging List?

Reply:
There is already `PagingList` from `graviti.manager.lazy`.